wasatch/FeatureIdentificationDevice.py

```diff
@@ -1397,6 +1321,89 @@ def get_high_gain_mode_enabled(self):
         self.settings.state.high_gain_mode_enabled = 0 != self.get_code(0xec, lsb_len=1, label="GET_HIGH_GAIN_MODE_ENABLED")
         return self.settings.state.high_gain_mode_enabled
 
+    ############################################################################
+    # Laser commands
+    ############################################################################
+
+    def get_opt_laser_control(self):
+        return self.get_upper_code(0x09, label="GET_OPT_LASER_CONTROL", msb_len=1)
+
+    def get_opt_has_laser(self):
+        available = (0 != self.get_upper_code(0x08, label="GET_OPT_HAS_LASER", msb_len=1))
```

Comment:
If I'm understanding this line correctly, to clean this up I would change this to bool(self.get_upper_code(....))